### ISSUE TITLE
User feedback

### ISSUE THREAD

**Issue by suryasriram** [shreevatsa/sanskrit#135]
I have an issue to report about the results for the following input:
```
viśuddha-vijñāna-ghanaṁ sva-saṁsthayā
samāpta-sarvārtham amogha-vāñchitam
sva-tejasā nitya-nivṛtta-māyā-
guṇa-pravāhaṁ bhagavantam īmahi
```

The issue is:
The first syllables of the third line svatejasā ... Namely 'sva' is a laghu syllable, but is being identified as guru, perhaps due to the samyuktakshara?

**Comment by shreevatsa** [comment]
Thanks for reporting! This was surprising, so I took a closer look… This is what I see for that input:

<img width="630" alt="image" src="https://user-images.githubusercontent.com/767530/178086894-a4abbaab-f373-49b9-9a12-e75d935cf9ef.png">

Actually what's happened is that it has correctly identified the `sva` as laghu (see the "Scanning the input" part of the debugging output, which I expanded in the above screenshot, or hover over the `sva`), but because it has stupidly matched a specific metre that happens to have G in that position (one of the "concrete" versions / "crystallizations" of Vaṃśastha), it is highlighting the syllable in red and pointing out that (if the poem was indeed intended to be specifically in "vaidhātrī" metre), then this is wrong.

The right solution here would be:

- identify the metre as a partial match for Vaṃśastha more generally (rather than "vaidhātrī" specifically), which allows that syllable to be either L or G,
- highlight only the missing syllable in __māyā__

I'd be happy with that but actually, this is a tricky case, because the metre is clearly "broken" in a sense, and yet (searching online) that seems to be the verse text… so maybe we should say lines 1, 2, 4 are in Vaṃśastha and line 3 is in Upajāti (specifically upendravajrā)?

**Comment by suryasriram** [comment]
Thank you for the reply! Understood to a certain extent why this error
occurred.

So it is then just an irregular Vamshastha with no particular name? Since
Bhagavatam is replete with many such types of shlokas.

Also I really appreciate your efforts in project, being a coder and a
Sanskrit enthusiast myself. It's a monumental effort from your side, which
will benefit students and scholars greatly.

Thanks and Regards,
Surya.

**Comment by shreevatsa** [comment]
Right, see the scansion (same as what the tool shows except I changed the very last syllable to G):

```
LGLGGLLGLGLG
LGLGGLLGLGLG
LGLGGLLGLGG
LGLGGLLGLGLG
```

— here lines 1, 2, 4 are in one metre and line 3 is in another. I've heard think such "mixtures" do occur in the literature… are there many more in the Bhagavatam?

(Thanks for your appreciation about the project… maybe it will give the motivation to return to this some day and fix remaining bugs / make improvements!)
